pulsar-client-cpp/python/setup.py

```diff
@@ -73,18 +73,24 @@ def build_extension(self, ext):
 
 dependencies = [
     'fastavro==0.24.0',
```

Comment:
Avro also is not a lightweight dep and it's only needed if someone wants to use Avro schema. It could be moved to its own extra section